packages/icons/package-lock.json

```diff
@@ -14,6 +14,7 @@
         "@storybook/theming": "^6.1.21",
         "@types/react": "16",
         "@types/react-dom": "16",
+        "loki": "^0.28.1",
```

Comment:
Why do we need to touch this package at all? Have we added any new icons?

Reply:
@aigoncharov  i have no idea why that changed at all. half of the time i have fought with npm here :/ there were many conflicts and what not so far; i stumbled upon a working version and stuck with it.

Reply:
@utkukaratas do you think we could revert this change as it introduces an absolutely unnecessary dependency to a package we do not even touch in this PR?

Reply:
will try for sure.. and i was afraid you're going to say that 😬

Reply:
Sorry :(

Reply:
@aigoncharov done. luckily it was not big deal; resolving.